spec/src/modules/browse.js

```diff
@@ -656,6 +675,24 @@ describe('ConstructorIO - Browse', () => {
       });
     });
 
+    it('Should return a response with valid ids and hiddenFields', (done) => {
+      const hiddenFields = ['hiddenField1', 'hiddenField2'];
+      const { browse } = new ConstructorIO({
+        ...validOptions,
+        fetch: fetchSpy,
+      });
+
+      browse.getBrowseResultsForItemIds(ids, { hiddenFields }).then((res) => {
+        const requestedUrlParams = helpers.extractUrlParamsFromFetch(fetchSpy);
+
+        expect(res).to.have.property('request').to.be.an('object');
+        expect(res).to.have.property('response').to.be.an('object');
+        expect(res).to.have.property('result_id').to.be.an('string');
```

Comment:
Might be missing an expect statement here:

```
expect(res.request.hidden_fields).to.eql(hiddenFields);```

Reply:
Good catch! Thank you